pkg/config/config.go

```diff
@@ -236,6 +236,10 @@ type PluginConfig struct {
 	// container requests with huge page limits if the cgroup controller for hugepages is not present.
 	// This helps with supporting Kubernetes <=1.18 out of the box. (default is `true`)
 	TolerateMissingHugetlbController bool `toml:"tolerate_missing_hugetlb_controller" json:"tolerateMissingHugetlbController"`
+	// DisableHugetlbController indicates to disable the hugetlb controller even when it is
+	// present in /sys/fs/cgroup/cgroup.controllers.
+	// This helps with running rootless mode + cgroup v2 + systemd but without hugetlb delegation.
```

Comment:
should mention `silently` here... and / or..  log a `one time?` warning at init or at first collision that support has been disabled by configuration.

Reply:
Added `silently`. When the config is present, the user wouldn't want the warning to be printed.

Reply:
it would be confusing... ok.